sdk/eventhub/event-hubs/src/util/connectionStringUtils.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { parseConnectionString } from "@azure/core-amqp";
+
+/**
+ * The set of properties that comprise an Event Hub connection string.
+ */
+export interface EventHubConnectionStringProperties {
+  /**
+   * The fully qualified Event Hub namespace extracted from the "Endpoint" in the
+   * connection string. This is likely to be similar to "{yournamespace}.servicebus.windows.net".
+   * This is typically used to construct the ServiceBusClient.
+   */
+  fullyQualifiedNamespace: string;
+  /**
+   * The value for "Endpoint" in the connection string.
+   */
+  endpoint: string;
+  /**
+   * The value for "EntityPath" in the connection string which would be the name of the queue or
+   * topic associated with the connection string.
+   * Connection string from a Shared Access Policy created at the namespace level
+   * will not have the EntityPath in it.
+   */
+  eventHubName?: string;
```

Comment:
Make it `entityPath` instead?
Since the users would see `EntityPath` in the connection string.

@ramya-rao-a

Reply:
Happy to change it of course, but just wanted to mention that the issue #11894 uses EventHubName to reflect the EntityPath token. Anyway, let me know what you folks think. Thanks for reviewing!

Reply:
I prefer keeping this as `eventHubName` since that will match the property name we have for our EventHubs client constructors. I think that's what was the motivating factor in #11894

Reply:
Thanks!